indigo_app/static/javascript/indigo/views/annotations.js

```diff
@@ -562,10 +570,9 @@
 
     scrollToElement: function(element) {
       const container = element.closest('.document-sheet-container');
-      const top = element.getBoundingClientRect().top;
       if (container) {
-        container.scrollBy({
-          top: top - container.getBoundingClientRect().top - 70,
+        container.scrollTo({
+          top: parseFloat(element.style.top.replace('px', '')) - 50,
```

Comment:
I found that this yields the most accurate results

Reply:
Nice, takes advantage of what the gutter has already calculated.